modules/effects/arrow-effects-data/src/main/kotlin/arrow/effects/IO.kt

```diff
@@ -56,7 +55,13 @@ sealed class IO<out A> : IOOf<A> {
             IO { callback(Left(t)) }
           }
 
-          IORunLoop.startCancelable(fa, conn2, mapUnit)
+          //DEV: If fa cancels conn2 like so `conn.cancel().map { cb(Right(Unit)) }`
```

Comment:
nit: space after //

Reply:
That sort of things should be reported by detekt / ktlint if we care enough